README.md

```diff
@@ -93,17 +93,18 @@ $ brew install carthage
 To integrate VirgilCryptoWrapper into your Xcode project using Carthage, create an empty file with name *Cartfile* in your project's root folder and add following lines to your *Cartfile*
 
 ```
-github "VirgilSecurity/virgil-cryptowrapper-x" ~> 0.15.2
+github "VirgilSecurity/virgil-cryptowrapper-x" ~> 0.16.0-rc1
```

Comment:
For production release it should be change to '~> 0.16.0'

Reply:
Fixed